src/commands/dev.ts

```diff
@@ -418,8 +419,42 @@ export async function command(commandOptions: CommandOptions) {
     }
   });
 
-  const server = http
-    .createServer(async (req, res) => {
+
+  const readCredentials = async () => {
+    const [cert, key] = await Promise.all([fs.readFile(__dirname + '/snowpack.crt'),fs.readFile(__dirname + '/snowpack.key')])
+    
+    return {
+      cert,
+      key,
+    }
+  };
+  
+  const certify = () =>
+  execa.sync(__dirname + '/certify.sh', {
```

Comment:
```
execa.sync(path.join(__dirname, '../assets/certify.sh')
```
Note that this is the correct path after building, when the entire package JS has been bundled into `pkg/dist-node/index.js`